lib/publiccloud/instance.pm

```diff
@@ -19,6 +19,9 @@ has instance_id => undef;               # unique CSP instance id
 has public_ip   => undef;               # public IP of instance
 has username    => undef;               # username for ssh connection
 has ssh_key     => undef;               # path to ssh-key for connection
+has image_id    => undef;               # image from where the VM is booted
+has region      => undef;
```

Comment:
we should have region in provider only

Reply:
Hm yes, you could access the region then with

```
$instance->provider->region
```